packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -83,112 +59,298 @@ describe('download', () => {
         'windows',
         'linux-musl',
       ],
-      version: CURRENT_BINARIES_HASH,
+      version: CURRENT_ENGINES_HASH,
     })
 
     // Check that all binaries git hash are the same
-    expect(await getVersion(queryEnginePath)).toContain(CURRENT_BINARIES_HASH)
-    expect(await getVersion(introspectionEnginePath)).toContain(CURRENT_BINARIES_HASH)
-    expect(await getVersion(migrationEnginePath)).toContain(CURRENT_BINARIES_HASH)
-    expect(await getVersion(prismafmtPath)).toContain(CURRENT_BINARIES_HASH)
-  })
+    expect(await getVersion(queryEnginePath)).toContain(CURRENT_ENGINES_HASH)
+    expect(await getVersion(introspectionEnginePath)).toContain(CURRENT_ENGINES_HASH)
+    expect(await getVersion(migrationEnginePath)).toContain(CURRENT_ENGINES_HASH)
+    expect(await getVersion(prismafmtPath)).toContain(CURRENT_ENGINES_HASH)
 
-  test('auto heal corrupt binary', async () => {
-    const platform = await getPlatform()
-    const baseDir = path.join(__dirname, 'corruption')
-    const targetPath = path.join(baseDir, getBinaryName('query-engine', platform))
-    if (fs.existsSync(targetPath)) {
-      try {
-        fs.unlinkSync(targetPath)
-      } catch (e) {
-        console.error(e)
-      }
-    }
-
-    await download({
-      binaries: {
-        'query-engine': baseDir,
-      },
-      version: FIXED_BINARIES_HASH,
-    })
-
-    fs.writeFileSync(targetPath, 'incorrect-binary')
-
-    // please heal it
-    await download({
-      binaries: {
-        'query-engine': baseDir,
-      },
-      version: FIXED_BINARIES_HASH,
-    })
-
-    expect(fs.existsSync(targetPath)).toBe(true)
-
-    expect(await checkVersionCommand(targetPath)).toBe(true)
-  })
-
-  test('handle non-existent binary target', async () => {
-    await expect(
-      download({
-        binaries: {
-          'query-engine': __dirname,
+    const files = getFiles(baseDir).map((f) => ({ ...f, size: 'X' }))
```

Comment:
Maybe just 
```suggestion
    const files = getFiles(baseDir).map((f) => f.name)
```
to make the inline snapshot shorter?

Just an idea, no strong preference either way.